includes/Core/Admin_Bar/Admin_Bar.php

```diff
@@ -206,7 +206,7 @@ public function is_active() {
 			}
 		} else {
 			// Otherwise use more general permission check (typically admin-only).
-			if ( ! current_user_can( Permissions::VIEW_DASHBOARD ) ) {
+			if ( ! current_user_can( Permissions::VIEW_ADMIN_BAR_MENU ) ) {
```

Comment:
Simply updating the capability checked here is not enough because it doesn't account for the other part of the condition which checks a different capability.

Here we can lift this check earlier because if someone doesn't have this capability, much of these checks can be skipped.

Reply:
I've pushed a fix for this 👍

Reply:
Thanks - makes sense! We don't want to show it regardless of the `VIEW_POST_INSIGHTS` check...